src/main/java/com/cloudbees/jenkins/GitHubPushTrigger.java

```diff
@@ -176,7 +177,13 @@ public void run() {
 
     private boolean createJenkinsHook(GHRepository repo, URL url) {
         try {
-            repo.createHook("jenkins", Collections.singletonMap("jenkins_hook_url", url.toExternalForm()), null, true);
+            if (getDescriptor().isUseWebHooks()) {
+                Collection<GHEvent> events = new ArrayList();
+                events.add(GHEvent.PULL_REQUEST);
```

Comment:
Sorry, failed on the event(I was also working on a separate plugin). I'll fix that.